skaffold.yaml

```diff
@@ -80,18 +80,13 @@ profiles:
   - op: replace
     path: /build/artifacts/7/docker/dockerfile
     value: Dockerfile.debug
-# "networkpolicy" profile adds network policies. By design, the default minikube cni flannel does not
-# support networkpolicies. Use an alternative CNI like calico
-# 'minikube start --cni=calico'
-#
-# This is not used by default. You can use it in isolation or in combination with other profiles:
-#     skaffold run -p networkpolicy, debug
-- name: networkpolicy
+# This profile is not used by default. You can use it in isolation or in combination with other profiles:
+#     skaffold run -p network-policies, debug
+- name: network-policies
```

Comment:
**My rationale for this change:**
`network-policies` matches the name of the Kustomize Component folder.
This consistency is good in case we end up turning more Kustomize Components into `skaffold` Profiles.